mods/mkrcraft/src/main/resources/mkrcraft.mixins.json

```diff
@@ -1,9 +1,12 @@
 {
-	"required": true,
-	"package": "pw.mkr.craft.mixins",
-	"compatibilityLevel": "JAVA_17",
-	"injectors": {
-		"defaultRequire": 1
-	},
-	"mixins": []
+  "required": true,
+  "package": "pw.mkr.craft.mixins",
+  "compatibilityLevel": "JAVA_17",
+  "injectors": {
+    "defaultRequire": 1
+  },
+  "server": [
+    "MinecraftServerMixin",
+    "PlayerEntityMixin"
+  ]
```

Comment:
Make these two changes to make IDEA use tabs.

![image](https://user-images.githubusercontent.com/19621208/150305203-4c4ca48f-cbe5-423c-ae7e-30b94942e67b.png)
![image](https://user-images.githubusercontent.com/19621208/150305449-23f3c42b-d7ee-4495-bb2c-dbe7fe12adba.png)